Mathlib/FieldTheory/PolynomialGaloisGroup.lean

```diff
@@ -422,6 +422,7 @@ theorem prime_degree_dvd_card [CharZero F] (p_irr : Irreducible p) (p_deg : p.na
   have hα : IsIntegral F α := .of_finite F α
   use FiniteDimensional.finrank F⟮α⟯ p.SplittingField
   suffices (minpoly F α).natDegree = p.natDegree by
+    letI _ : AddCommGroup F⟮α⟯ := Ring.toAddCommGroup
```

Comment:
Can you add a comment saying that sooner or later this line should be removed?

Reply:
Maybe we should create issue to track this problem of failing to find weaker instances from strong instances on subobjects, so we can refer to the issue here and elsewhere, which will make it easy to remove all these `letI/haveI`s once the issue is resolved?